ui/src/App.jsx

```diff
@@ -42,10 +35,8 @@ class App extends Component {
 
   sendUpdate() {
     const state = JSON.parse(JSON.stringify(this.props.state));
-    for (let sp of state.Subpolicies) {
-      sp.Metadata = JSON.parse(sp.Metadata);
-    }
-    axios.post(`${window.config.env.REACT_APP_NOPAS_ENDPOINT}/update`, state);
+    const out = uiToServerConversion(state);
+    out && axios.post(`${window.config.env.REACT_APP_NOPAS_ENDPOINT}/update`, out);
```

Comment:
Can use `new URL` to join paths in future PRs, e.g. ref: https://github.com/datagovsg/lauth-go/blob/master/ui/src/components/Login.js#L37